### PR TITLE
remove Track and IsTxFinal

### PR DESCRIPTION
**Pull request by adecaro**
This PR does remove the following view commands:
- `Track`. It was not implemented and never requested.
- `IsTxFinal`. Each application should provide this functionality only if needed.

### CODE REVIEW

File: integration/fabric/iou/views/finality.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
+/*
```
Comment (alexandrosfilios) [inline integration/fabric/iou/views/finality.go:1]: Maybe we could reuse the view that we defined before?

**Review by alexandrosfilios — APPROVED** [review]
LGTM modulo comment

File: integration/fabric/iou/views/finality.go (hunk 1 above)
Comment (adecaro) [reply integration/fabric/iou/views/finality.go]: done

**Review by alexandrosfilios — APPROVED** [review]
LGTM

**Pull request merged**